reference/release-notes/1.21.0.md

```diff
@@ -0,0 +1,67 @@
+* Production support for Vulkan on NVIDIA GPUs is available:
+  - The default graphics driver stack used is Vulkan, VirGL/Venus and ANGLE. Latency and flickering is improved for VirGL by introducing a dedicated component, frame renderer. <!--AC-2152-->
+  - System tracing is enabled for Android 12 and 13. This helps in improving the performance of your application. See [System tracing overview](https://developer.android.com/topic/performance/tracing) for more information. <!--AC-2161-->
+
+* In the [AMS HTTP API](https://anbox-cloud.github.io/latest/ams/), `/1.0` endpoint exposes the cluster ID to enable identifying a subcluster.<!--AC-2148-->
+
+### Streaming improvements
+
+< List enhancements to streaming stack here. >
+
+### Dashboard improvements
+
+The following improvements are available in the Anbox Cloud dashboard for this release:
+
+* Custom instance creation
+* Support multiple subclusters from a single dashboard for large scale deployments
+* Experimental "upscaling" parameter in session view
+* Experimental support for managing the Vehicle Hardware Abstraction Layer (VHAL) properties
+* Enhancements for better user experience
+* Enhancements for improved responsiveness, error handling and messages
+* Updated Python and Javascript dependencies.
+
+### Other
+
+* Android Automotive (AAOS) images with experimental VHAL support are available. See [Provided images](https://anbox-cloud.io/docs/reference/provided-images) for more details.
+* Android security updates for February 2024 (see [Android Security Bulletin - February 2024](tbd) for more information).<!--AC-2249-->
+* The Android WebView has been updated to [121.0.6167.71](https://chromereleases.googleblog.com/2024/01/chrome-for-android-update_17.html).
+
+## Removed functionality
+
+* Clustering support (which existed as an experimental feature) for Anbox Cloud Appliance is removed.<!--AC-2176-->
+* The deprecated Android 11 images are now unsupported and hence removed.<!--AC-2108-->
+
+## Deprecations
+
+< List deprecated features. Provide an alternative, if any, for the users to use instead of the deprecated features >
+
+## Known issues
+
+< List known issues with the current release. Include plans to fix it, if known. >
+
+## Bug fixes
+
+* `Invalid session` error When streaming an audio-only WebRTC session.<!--AC-2256-->
+* Initialising Anbox Cloud Appliance as root user causes a `missing AMC client cert` error. <!--AC-2255-->
+* [LP 2049734](https://bugs.launchpad.net/anbox-cloud/+bug/2049734) `INSTANCE_TYPE` is set to `regular` instead of `base` during the runtime of the post-stop hook of an addon.<!--AC-2238-->
+* Screen recording from the quick settings fails to start. This failure happens consistently on all Android versions, irrespective of the use of a GPU and software rendering. <!--AC-2180-->
```

Comment:
Let's drop this as the fix has been landed in the 1.20.2 release.
https://discourse.ubuntu.com/t/anbox-cloud-1-20-2-has-been-released/41686

Reply:
Thanks for the catch!